doc/adrs/XXX_support_representation_details_view.adoc

```diff
@@ -0,0 +1,50 @@
+= ADR-XXX - Add Support for Representation Details View
+
+
+== Context
+
+Currently, when a user selects a representation, nothing is displayed in the details section.
+Representations only have two metadata fields (name and kind).
+Users want more information to distinguish their different representations.
+
+== Decision
+
+To address this need, we will enrich the metadata of representations by adding additional information such as a description and an icon.
+These details will be displayed in the details section when a representation is selected.
+
+=== View DSL
+
+* Enhance the `representationDescription` type to add a description to the representation and the ability to define icons
+
+class RepresentationDescription {
+    EString description
+    InterpretedExpression iconURLsExpression
```

Comment:
Don't mention URL in the expression since for example, the node image does not have to be an URL.